Agriculture Project/Readme.md

```diff
@@ -1 +1,112 @@
+<h1 align="center">INDIAN CROP RECOMENDATION SYSTEM</h1> 
+<img src="India-promising.jpg" width="1500" height="800">
 
+# Aim of the project - 
+-  Agriculture is an important part of India's economy and at present it is among the <b>top two</b> farm producers in the world. This sector provides approximately <b>52 percent</b> of the total number of <b>jobs available in India</b> and contributes around <b>16 percent</b> to the GDP. Agriculture is the only means of living for almost two-thirds of the employed class in India. As being stated by the economic data of financial year 2006-07, agriculture has acquired <b>18 percent of India's GDP</b>. The agriculture sector of India has occupied almost <b>43percent of India's geographical area</b>.
+-  Agriculture in India plays a predominant role in economy and employment. The common problem existing among the Indian farmers are they don’t choose the right crop based on their soil requirements. Due to this they face a serious setback in productivity. if they can choose right crop for the cultivation based on soil and environmental factores yeild of the crop will be increases and famers can get better profits.
+- This model is bulit to solve this problem as much as possible and hope this model will help farmers in selecting main crop which is suitable for the soil.
+
+# Data
+
+- <b>SOURCE</b> 
+    - Data hase been taken from the <a href="https://www.kaggle.com/atharvaingle/crop-recommendation-dataset">Kaggle : Kaggle Crop recomendation dataset</a> ,  <b>Data contains 7 coloumns</b> 
+<img src="data coloumns.png" width="800" height="300">
+
+- <b> Data overview</b>- 
+    - columns like N P and K are renamed as Nitrogen Phosphoroous and Potassion
+ ![](2021-03-15-17-16-04.png)
+    - There are no null values in the data 
```

Comment:
Add a section as Outier analysis and then just write that there were no outlier at each column level, also explain how outlier analysis was done